aidies-project/client/src/components/LoginPanel.js

```diff
@@ -1,45 +1,117 @@
 import React, { useState } from 'react';
-import { Redirect, NavLink } from 'react-router-dom';
+import { Redirect, NavLink, useHistory } from 'react-router-dom';
 import { useDispatch, useSelector } from 'react-redux';
 import { login } from '../store/authentication';
-// import { Redirect } from 'react-router-dom';
+
+import {Button, 
+        CssBaseline, 
+        TextField, 
+        Link, 
+        Grid, 
+        Typography, 
+        Container} from '@material-ui/core';
```

Comment:
Super knit-picky comment about import formatting.
```suggestion
import {
    Button, 
    CssBaseline, 
    TextField, 
    Link, 
    Grid, 
    Typography, 
    Container
} from '@material-ui/core';
```